packages/bar/src/BarCanvas.tsx

```diff
@@ -32,6 +33,13 @@ import { renderAxesToCanvas, renderGridLinesToCanvas } from '@nivo/axes'
 import { renderLegendToCanvas } from '@nivo/legends'
 import { useTooltip } from '@nivo/tooltip'
 import { useBar } from './hooks'
+import {
+    updateGreatestValueByIndex,
+    updateNumberOfBarsByIndex,
+    updateTotalsByIndex,
+    updateTotalsPositivesByIndex,
+} from './BarTotals'
```

Comment:
Imports could be defined in a separate file, so that we don't have to import an SVG component for canvas.